### ISSUE TITLE
Zooming should reset auto-centre timer

### ISSUE THREAD

**Issue by mikehgentry** [osmandapp/OsmAnd#15306]
Set the auto-centre timer to a low value like 5 seconds. Pan around the map for a bit and it works as expected. Stop panning and start zooming using the +/- buttons, and the map auto-centres, even though the user is still interacting with the viewport.

### Your Environment
OsmAnd Version: + 4.2.7
Android/iOS version: 11
Device model: XCover 4s

**Comment by vshcherb** [comment]
Indeed Autocentre shouldn't happen if you just changed the zoom

**Comment by Zirochkabila** [comment]
# Testing 1
`OsmAnd~ 5.1.0#5216m, Released: 2025-05-16`

@mikehgentry Automatic panning using different time intervals (5, 15, 25) has been tested - when the user interacts with the court and uses the “+”, “-” buttons to pan, the map is not centered on the navigation. The time to centering is counted only after the user has stopped interacting (touching) with the map

**Comment by mikehgentry** [comment]
I tried this on nightly version 5.1.0#522m,, 2015-5-18, still not working as expected as far as I can see? I mean feel free to close as wontfix, it's a minor bug, but it isn't fixed I don't think...

**Comment by mikehgentry** [comment]
Possibly part of #15307 anyway

**Comment by mikehgentry** [comment]
Sorry for the comment spam, but maybe there's a language barrier at work here.

What I'm saying is the map *shouldn't* auto-centre if you're pressing the +/- buttons. This might not seem relevant in artificial testing scenarios, but if you're out in the wilderness it matters - if you're looking at an area that is still downloading, and you hit zoom a couple of times, it might well jump back to where you are. I think this is contrary to user expectation - a zoom is no different to a pan.

**Comment by Zirochkabila** [comment]
@mikehgentry As far as I can see, the map is not automatically centered when the user interacts with the pan or zoom buttons.
If you have a different result, please add a video with the steps (actions) and the result + make sure you are using the night version.

**Comment by dmpr0** [comment]
# Ui Review 

- [ ] Auto-center map does not account for tapping on zoom buttons.

**Comment by Zirochkabila** [comment]
# Testing 2
`OsmAnd~ 5.1.0#5293m, Released: 2025-05-30`

The map centering time after pressing “+” “-” is checked - **The map is centered after the set time (in the video - 10 seconds)** 

<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/8ab20ca7-86ec-4f7e-b3a4-e01d7da94d93"></video>
</details>
